eng/scripts/Language-Settings.ps1

```diff
@@ -47,5 +47,7 @@ function Publish-c-GithubIODocs ($DocLocation, $PublicArtifactLocation)
     # used to publish multiple docs packages in a single invocation.
     $pkgInfo = Get-Content $DocLocation/package-info.json | ConvertFrom-Json
     $releaseTag = RetrieveReleaseTag "C" $PublicArtifactLocation
-    Upload-Blobs -DocDir $DocLocation -PkgName 'docs' -DocVersion $pkgInfo.version -ReleaseTag $releaseTag
+    Upload-Blobs -DocDir $DocLocation -PkgName 'az_core' -DocVersion $pkgInfo.version -ReleaseTag $releaseTag
+    Upload-Blobs -DocDir $DocLocation -PkgName 'az_iot' -DocVersion $pkgInfo.version -ReleaseTag $releaseTag
+    Upload-Blobs -DocDir $DocLocation -PkgName 'az_storage_blobs' -DocVersion $pkgInfo.version -ReleaseTag $releaseTag
```

Comment:
@sima-zhu, can you help me understand what is the desired end result of this change?

Reply:
The end result is to essentially mimic 3 packages even though there is really only 1. This is align the c repo with other languages when we look at the package index. By duplicating these github.io docs it makes it look like there are 3 packages even though there is really only one.

Reply:
Thanks @weshaggard helping answer this. I missed the comments.